src/java.base/share/classes/java/util/concurrent/Executors.java

```diff
@@ -392,25 +389,19 @@ public static ThreadFactory defaultThreadFactory() {
 
     /**
      * Returns a thread factory used to create new threads that
-     * have the same permissions as the current thread.
+     * have current context class loader as the context class loader.
```

Comment:
One nit for your consideration: "...to create new threads that have **_the_** current context class loader..."

Reply:
Fixed in https://github.com/openjdk/jdk/pull/21498/commits/06c4c3c1ab1fe121b625bd30a0c424be06d5022a